packages/core/src/external/aws/s3.ts

```diff
@@ -356,41 +356,6 @@ export class S3Utils {
     }
   }
 
-  async retrieveDocumentIdsFromS3(
```

Comment:
Moving this function out of `external/aws/s3`, as discussed here - [context](https://metriport.slack.com/archives/C04DMKE9DME/p1719274157562819?thread_ts=1718897333.131819&cid=C04DMKE9DME)